tools/fq.wdl

```diff
@@ -86,10 +83,6 @@ task fqlint {
             ~{"'" + read_two_fastq + "'"}
     >>>
 
-    output {
-        String check = "passed"
-    }
-
     runtime {
         memory: "~{memory_gb} GB"
         disks: "~{disk_size_gb} GB"
```

Comment:
Not this line, but the `subsample` task has a 4GB of memory. Do we need that or is the spec default of 2 sufficient?